### PR TITLE
Speak monster spawner mob type

### PR DESCRIPTION
**Pull request by boholder**
Request from Trix from Discord

### CODE REVIEW

File: common/src/main/resources/minecraft_access-common.mixins.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
   "package": "com.github.khanshoaib3.minecraft_access.mixin",
   "compatibilityLevel": "JAVA_17",
   "mixins": [
```
Comment (khanshoaib3) [inline common/src/main/resources/minecraft_access-common.mixins.json:6]: Just a small refactor, these are better to be moved to "client" section

**Review by khanshoaib3 — APPROVED** [review]

File: common/src/main/resources/minecraft_access-common.mixins.json (hunk 1 above)
Comment (boholder) [reply common/src/main/resources/minecraft_access-common.mixins.json:6]: I've noticed that, but I can't find a valiable field for extracting the mob name from pure-client-side classes.

Comment (boholder) [reply common/src/main/resources/minecraft_access-common.mixins.json:6]: > Just a small refactor, these are better to be moved to "client" section

I've noticed that, but I can't find a valiable field for extracting the mob name from pure-client-side classes.
Oh, you mean we can simply move the two to `client` list without any error?

Comment (khanshoaib3) [reply common/src/main/resources/minecraft_access-common.mixins.json:6]: Yup, that should not cause any error and ensure that the mod is client sided.

**Pull request merged**